tutor/src/components/task-plan/lms-info.jsx

```diff
@@ -137,7 +137,6 @@ export class LmsInfoLink extends React.PureComponent {
   }
 
   render() {
-    if (!this.getStats().shareable_url) { return null; }
 
```

Comment:
this is the bug - sometimes the BE doesn't generate a shareable_url for preview courses